### ISSUE TITLE
Can't update chart to 3d if xAxis.categories is present

### ISSUE THREAD

**Issue by DiegoCardoso** [highcharts/highcharts#8555]
#### Expected behaviour
If I have a 2d chart with one `xAxis` with `categories` set and try to update it to became a 3d chart by, for instance, calling 
```js
	chart.update({
		chart: {
			options3d: {enabled: true, /* other values */}
		}
	})
```

I want the chart to be redraw using the 3d configuration provided.

#### Actual behaviour
Chart is not able to complete the redraw process, series become blank and an error `Uncaught TypeError: Cannot read property 'left' of undefined` is raised on browser console.

The error happens in [this line](https://github.com/highcharts/highcharts/blob/v6.1.1/js/parts-3d/Axis.js#L589), which is trying to read `chart.frameShapes` that seems to not be generated if `options3d.enabled` is not set on the first draw.

#### Live demo with steps to reproduce
http://jsfiddle.net/pt5emc9d/3/

1. Open jsfiddle
2. Click on the button to update configuration
3. Repeat the process, now with HC 6.1.0 instead to see it working.

#### Product version

Highcharts/Highstock 6.1.1
#### Affected browser(s)
Tested on Chrome

**Comment by pawelfus** [comment]
Hi @DiegoCardoso 

Thank you for reporting this regression. 

*Internal note:*
Bisect leads to 1477eac80 @Denyllon

**Comment by alvarezguille** [comment]
Looks like this was fixed in `6.1.2`

**Comment by pawelfus** [comment]
Thanks @alvarezguille ! you are right, fixed since v6.1.2, closing the issue then.
